fs/info.py

```diff
@@ -82,23 +82,18 @@ def _make_datetime(self, t):
         else:
             return None
 
-    @overload
-    def get(self, namespace, key, default=None):  # pragma: no cover
-        # type: (Text, Text, Optional[T]) -> Optional[T]
-        pass
-
     @overload
     def get(self, namespace, key):  # pragma: no cover
-        # type: (Text, Text) -> Optional[T]
+        # type: (Text, Text) -> Any
```

Comment:
Mypy wasn't having this. The return type was more general than the original code implied.

It would actually me more accurate to replace Any with a union of str, int, list, dict etc, but that would require a cast everywhere `get` is used. Could still be persuaded to do that...